docs/faq/troubleshooting/why-is-my-file-missing.md

```diff
@@ -34,7 +34,7 @@ As a solution to this problem, we placed a size limit to the files that Codacy w
 
 ### What if I need to analyze a file that exceeds this limit?
 
-While Codacy will discard your file by default, you can still have it analyzed using the [Codacy Analysis CLI](../../related-tools/running-local-analysis.md) and running the analysis locally. CLI doesn't have a set timeout or any limitation on file size, so you will be able to have your files analyzed this way.
+While Codacy will discard your file by default, you can still have it analyzed using the [Codacy Analysis CLI](../../related-tools/local-analysis/running-local-analysis.md) and running the analysis locally. CLI doesn't have a set timeout or any limitation on file size, so you will be able to have your files analyzed this way.
```

Comment:
`CLI doesn't have a set timeout or any limitation on file size` -> this is not 100% true - we do not have a timeout for the total analysis, but we do have a timeout for each tool execution. Maybe we can be clearer here on this or just not refer timeouts

Reply:
Well spotted @pedrocodacy! :mag: 

I removed the reference to the timeout since, in this context, it's precisely the time of each tool execution that could prevent the analysis from being successful. :thinking: